base/debian/changelog

```diff
@@ -4,6 +4,13 @@ cuttlefish-common (1.4.0) UNRELEASED; urgency=medium
 
  -- Chad Reynolds <chadreynolds@google.com>  Fri, 04 Apr 2025 13:05:10 -0700
 
+cuttlefish-common (1.3.1) stable; urgency=medium
```

Comment:
should this be only part of the v1.3.0 branch? I'm afraid that if a user checks out the main branch at this specific commit 985be6, it won't reproduce the right v1.3.1  package.

Reply:
I've squashed the commits together so there is no longer a commit with 1.3.1 at the top and a mismatched repository state.